samples/app/FilamentApp.cpp

```diff
@@ -329,13 +333,31 @@ void FilamentApp::run(const Config& config,SetupCallback setupCallback,
 
         // TODO: we need better timing or use SDL_GL_SetSwapInterval
         SDL_Delay(16);
+
         Renderer* renderer = window->getRenderer();
+
+        if (render) {
```

Comment:
I wouldn't bother with the `if` here since we're providing a default no-op function.

Reply:
The default is safe but the caller could pass nullptr.